tests/e2e/test.sh

```diff
@@ -6,6 +6,9 @@ set -xe
 HOST=127.0.0.1
 PORT=30080
 BASE_URL="http://$HOST:$PORT/v1"
+REPLICAS=${REPLICAS:-3}
+REQUESTS=60
```

Comment:
Should this be read from env as well? I saw some setup in `tests.yml`

Reply:
Yes it should. Thanks for catching this!

Reply:
Done